roles/cluster_ensure_machineset/tasks/derive_machineset.yml

```diff
@@ -6,9 +6,20 @@
   register: oc_get_machinesets
   failed_when: not oc_get_machinesets.stdout
 
+- name: Set base_machineset
+  command:
+    echo {{ oc_get_machinesets.stdout[0] }}
+  register: base_machineset
+  when: not base_machineset 
```

Comment:
Please declare `base_machineset` in the [`defaults/main/config.yml` file](https://github.com/openshift-psap/ci-artifacts/blob/8b6bb0136d21d333367b12c33d05375503ef60e8/roles/cluster_set_scale/defaults/main/config.yml#L1)